src/geodesic.rs

```diff
@@ -2258,10 +2258,9 @@ mod tests {
     }
 
     #[test]
-    #[ignore] // Fails existing behavior.
     fn test_std_geodesic_geodsolve29() {
         // Check longitude unrolling with inverse calculation 2015-09-16
-        let geod = Geodesic::new(6.4e6, 0.1);
+        let geod = Geodesic::wgs84();
         let (_a12, s12, _salp1, _calp1, _salp2, _calp2, _m12, _M12, _M21, _S12) =
             geod._gen_inverse(0.0, 539.0, 0.0, 181.0, caps::STANDARD);
         // Note: This is also supposed to check adjusted longitudes, but geographiclib-rs
```

Comment:
I guess this portion of the test is still TODO. 

By "adjusted longitude" - is that the same thing as feature enabled by the `LONG_UNROLL` flag, or is this something else?

Reply:
Right, that part is still todo. It's checking an output that's included in Java, but not in C++. I don't remember how closely it's related to the LONG_UNROLL flag, but it comes down to providing adjusted versions of some inputs as outputs. Of course, it's possible that I'm currently missing a straightforward way to request the comparable value in C++ -- I haven't taken a very close look. Regardless, I don't expect to revisit that any time in the immediate future. We've got lots of bigger issues to chase down first.